pkg/deprecatetemplates/allowlist.go

```diff
@@ -0,0 +1,90 @@
+package deprecatetemplates
+
+import (
+	"io/ioutil"
+	"os"
+	"sync"
+
+	"github.com/ghodss/yaml"
+	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+type deprecatedTemplateBlocker struct {
+	Jira        string      `json:"jira"`
+	Description string      `json:"description"`
+	Jobs        sets.String `json:"jobs"`
```

Comment:
Can this be a slice so the serialized form does not look so silly?

Reply:
Not really. I want to add some per-job information to the allowlist (mostly if the job is handcrafted or generated), so it is obvious from the file.

Reply:
Err either this is a list of jobs (which can be deduped on load) or it's a mapping of job to something, and not a set. I am just saying don't use the set type since the serialization looks bad.